File: app/src/main/java/com/rachelleignacio/listbucket/domain/interactors/DeleteListItemInteractor.java
package com.rachelleignacio.listbucket.domain.interactors;

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor;

/**
 * Created by rachelleignacio on 3/10/17.
 */

public interface DeleteListItemInteractor extends Interactor {
    interface Callback {
        void onListItemDeleted();
    }
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/domain/interactors/DeleteListItemInteractor.kt
package com.rachelleignacio.listbucket.domain.interactors

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor

/**
 * Created by rachelleignacio on 8/9/17.
 */
interface DeleteListItemInteractor : Interactor {
    interface Callback {
        fun onListItemDeleted(adapterPosition: Int)
    }
}